photo.html

```diff
@@ -119,6 +119,30 @@ <h2 class="visually-hidden">Галерея пользователей</h2>
           <p class="gallery__baner-header">Затмите их всех! Выкладывайте все, что накопилось в телефоне!</p>
         </div>
       </section>
+      <section class="post">
+        <img class="post__img" src="" alt="" width="" height="">
+        <form class="post__filter">
+          <ul class="post__list">
+            <li class="post__item">
```

Comment:
```suggestion
            <li class="post-upload__filter-item">
```

И в местах ниже, поправь пожалуйста название классов, чтобы было более понятно что происходит в этом блоке.